colrv1/all-compat.toml

```diff
@@ -0,0 +1,3705 @@
+
+family = "Noto Color Emoji Compat Test"
+output_file = "NotoColorEmojiCompatTest-Regular.ttf"
+color_format = "glyf_colr_1_and_picosvg_and_cbdt"
```

Comment:
as of nanoemoji v0.13 we don't support this hybrid color_format any more, only through maximum_color script, I suggest removing this all-compat.toml file

Reply:
Good catch, removing.